.github/workflows/create-github-release.yml

```diff
@@ -0,0 +1,32 @@
+name: create-github-release
+
+on:
+  push:
+    branches:
+      - main
+      - prerelease/**
+    tags-ignore:
+      - '*'
+  workflow_dispatch:
+    inputs:
+      prerelease:
+        type: string
+        description: 'Name to use for the prerelease: beta, dev, etc. NOTE: If this is already set in the package.json, it does not need to be passed in here.'
+
+jobs:
+  release:
+    uses: salesforcecli/github-workflows/.github/workflows/create-github-release.yml@main
+    secrets: inherit
+    with:
+      prerelease: ${{ inputs.prerelease }}
+      # If this is a push event, we want to skip the release if there are no semantic commits
+      # However, if this is a manual release (workflow_dispatch), then we want to disable skip-on-empty
+      # This helps recover from forgetting to add semantic commits ('fix:', 'feat:', etc.)
+      skip-on-empty: ${{ github.event_name == 'push' }}
+  # docs:
+  #   # Most repos won't use this
+  #   # Depends on the 'release' job to avoid git collisions, not for any functionality reason
+  #   needs: release
+  #   secrets: inherit
+  #   if: ${{ github.ref_name == 'main' }}
+  #   uses: salesforcecli/github-workflows/.github/workflows/publishTypedoc.yml@main
```

Comment:
Looks like you'll want to uncomment the docs section since it existed in `onPushToMain.yml`